packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -52,6 +52,11 @@ export class VaButton {
    */
   @Prop() label?: string; // could use this.el.getAttribute('aria-label') but this is more explicit
 
+  /**
+   * If `true`, the button will use the primary alternate variant.
+   */
+  @Prop({ reflect: true }) primaryAlternate?: boolean = false;
```

Comment:
Did you need to make this reflective because it wasn't working without it in a parent component or something?

Reply:
I don't know, I just copied from the secondary prop. I removed it and it seems fine.